packages/uui-input/lib/uui-input.element.ts

```diff
@@ -158,6 +158,15 @@ export class UUIInputElement extends FormControlMixin(
       input::placeholder {
         transition: opacity 120ms;
       }
+
+      input[type='password']::-ms-reveal {
+        display: none;
+      }
+
+      input[type='password']::-ms-reveal {
```

Comment:
@BatJan looks good, but please remove one of these declarations. No need for two of the same :-P

Reply:
Haha! Ageeed! I'll get to it 😊